### PR TITLE
bls 0.10.1

### PR DESCRIPTION
**Pull request by mpetrunic**
- added new methods
- updated spec tests dep
- update bls spec tests to new test case format

TODO:
- ~~3 fast aggregate verify tests are failing~~

### CODE REVIEW

File: packages/bls/src/index.ts [hunk 1]
```diff
@@ -78,18 +77,36 @@ export function aggregatePubkeys(publicKeys: BLSPubkey[]): BLSPubkey {
...
+export function verifyAggregate(publicKeys: BLSPubkey[], messageHash: bytes32, signature: BLSSignature): boolean {
+  assert(publicKeys, "publicKey is null or undefined");
+  assert(messageHash, "messageHash is null or undefined");
+  assert(signature, "signature is null or undefined");
+  try {
+    return Signature
+      .fromCompressedBytes(signature)
+      .verifyAggregate(publicKeys, messageHash);
   } catch (e) {
+    console.log(e)
```
Comment (mpetrunic) [inline packages/bls/src/index.ts]: @wemeetagain Do you think it makes sense to introduce "debug" dependency for ssz and bls so we can debug when other people are using it?

**Comment by mpetrunic** [comment]
@ChainSafe/eth2-0 we cannot pass CI until everything is updated to 0.10.x spec but we are passing bls stuff which should be enough to release it

File: packages/bls/src/index.ts (hunk 1 above)
Comment (wemeetagain) [reply packages/bls/src/index.ts]: possibly? Curious why this would throw in the first place?
Perhaps we can just bubble up errors

Comment (mpetrunic) [reply packages/bls/src/index.ts]: Well sometimes invalid serialization, wrong order, there is bunch of things though. Something wrong with messages, length etc

Comment (wemeetagain) [reply packages/bls/src/index.ts]: I feel like those things could trigger an error rather than a false.

Comment (mpetrunic) [reply packages/bls/src/index.ts]: well invalid signature order should trigger false though

Comment (mpetrunic) [reply packages/bls/src/index.ts]: Let's release like this. Invalid signatures in spec tests are breaking everything if we remove try/catch

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**